index.js

```diff
@@ -43,16 +43,17 @@ function checkGuess() {
     if (guess < targetNumber) {
       tooLowMessage.style.display = '';
     } else {
```

Comment:
If you input blank, it thinks the number is too high.   Would have checked that the input was in the valid range.

Reply:
added a commit to fix the non input bug!